go.mod

```diff
@@ -59,7 +60,7 @@ require (
 	github.com/elastic/gosigar v0.12.0 // indirect
 	github.com/felixge/httpsnoop v1.0.1 // indirect
 	github.com/filecoin-project/go-cbor-util v0.0.0-20191219014500-08c40a1e63a2 // indirect
-	github.com/filecoin-project/go-data-transfer v1.15.1 // indirect
+	github.com/filecoin-project/go-data-transfer v1.15.2-0.20220615120701-59f73123719f // indirect
```

Comment:
Depends on a fix that would have otherwise caused a panic with the latest ipld-prime dependency, fixed here: https://github.com/filecoin-project/go-data-transfer/pull/335